### DISCUSSION TITLE
Mainsail invalid after update

### DISCUSSION THREAD

**Discussion by JPoGger** [mainsail-crew/mainsail#1712] [Help]
### Mainsail Version:

v2.9.0

### Browser:

Other (Please tell us below)

### Device:

Laptop

### Operating System:

Windows

### What happened?

I updated several things mainsail, moonraker etc... then it took mainsail several minutes to initialize finally it initialized then I checked in on machine tab, and saw this. Already rebooted my machine and updated Ubuntu.
![image_2023-12-23_213251341](https://github.com/mainsail-crew/mainsail/assets/128104933/49f63ffd-c2f4-4b49-9be6-d8c144d690a0)

### What did you expect to happen instead?

I expected it to have successfully updated.

### How to reproduce this bug?

I don't completely know this could have been from a bad internet connection while updating though.

### Additional information:

- Was using Opera GX

- Wasn't using a pi for klipper was using an old laptop running Ubuntu

- Used KIAH to set it up

**Comment by meteyou** [comment]
@JPoGger i moved your issue to the discussions, because its not a bug from mainsail. We only use the issue tracker for bugs & features.

pls check your moonraker config. you have a typo in this config file:
![image](https://github.com/mainsail-crew/mainsail/assets/8167632/70baa215-0b36-45a7-8fec-5b480ef8871a)

it looks like you have a `]` at the end of the path.
